src/Models/Form.php

```diff
@@ -6,148 +6,156 @@
 
 class Form extends Model
 {
-	/**
-	* =========================
-	*		 TRAIT
-	* =========================
-	*/
+    /**
+     * =========================
+     *		 TRAIT
+     * =========================.
+     */
 
     //---------------------------
 
-	/**
-	* =========================
-	*		 CONSTANTS
-	* =========================
-	*/
-	
-	const FORM_ID                   = 'form_id';
-	const FROM_NAME                 = 'form_name';
-	const FORM_SLUG                 = 'form_slug';
-    const FORM_DESCRIPTION          = 'form_description';
-    const FORM_CONFIRMATION_MESSAGE = 'form_confirmation_message';
-    const IS_ACTIVE                 = 'is_active';
-    const FORM_MODEL                = 'form_model';
-    const FORM_CONTENT              = 'form_content';
-    const FORM_VERSION              = 'form_version';
-	const CREATED_AT                = 'created_at';
-	const UPDATED_AT                = 'updated_at';
-	const PRIMARY_KEY               = 'form_id';
-
-	/**
-	* =========================
-	*		 FIELDS
-	* =========================
-	*/
-
-	protected static string $tableName;
-	protected $table;
-	protected $primaryKey   = self::PRIMARY_KEY;
-	public $incrementing    = true;
-	protected $dateFormat   = 'Y-m-d';
-	
-	/**
-	* =========================
-	*		 CASTS
-	* =========================
-	*/
-
-	protected $casts = [
-		self::IS_ACTIVE     => 'boolean',
-        self::FORM_CONTENT  => 'json',
-	];
-	
-	/**
-	* =========================
-	*		 FILLABLE
-	* =========================
-	*/
-
-	protected $fillable = [
-		self::FROM_NAME,
+    /**
+     * =========================
+     *		 CONSTANTS
+     * =========================.
+     */
+    public const FORM_ID = 'form_id';
```

Comment:
This was Laravel `pint` doing its thing, but in the process it lost some of its formatting